patches/4.4/x86_64/0001-x86-build-Mark-per-CPU-symbols-as-absolute-explicitl.patch

```diff
@@ -0,0 +1,78 @@
+From d071ae09a4a1414c1433d5ae9908959a7325b0ad Mon Sep 17 00:00:00 2001
```

Comment:
I'd personally prefer the full set of patches because without them, the vDSO isn't being linked with LLD. These are the ones I sent to Sasha + the one you have that was already queued up: https://del.dog/raw/qocucuqume

`curl -LSso patches/4.4./x86_64/lld.patch https://del.dog/raw/qocucuqume`